docker/java.docker

```diff
@@ -31,42 +31,62 @@ RUN mv groovy-2.4.10 groovy
 ENV GROOVY_HOME /usr/local/groovy/
 ENV PATH /usr/local/groovy/bin:${PATH}
 
-# Setup GroovyServ
```

Comment:
GroovyServ is still downloaded in lines 25-27.
Is Groovy still needed (lines 29-32)?